port/raspberrypi/rp2xxx/src/hal/irq.zig

```diff
@@ -139,3 +169,48 @@ pub fn disable(mask: Mask) void {
         },
     }
 }
+
+const VtorFunction = *const fn () callconv(.C) void;
```

Comment:
We'd like it is this calling convention were not enforced, since we might want to write functions with `callconv(.naked)` for hand-written assemblers.